src/components/manage/UniversalLink/UniversalLink.jsx

```diff
@@ -100,7 +100,7 @@ const UniversalLink = ({
             : null
         }
         rel="noopener noreferrer"
-        className={className}
+        className={`${className ?? ''} external`}
```

Comment:
This is better and less error prone with `cx` from `classnames` lib

Reply:
No, I incorporated the changes.